### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by hirasso**
I happened to need this functionality for a current project, so I applied some cleanup. Shouldn't contain any breaking changes :)

### CODE REVIEW

**Comment by daun** [comment]
Have you run npm update with this one? Might be good too. Otherwise, looks fine!

**Comment by daun** [comment]
(Can't approve as it isn't assigned to me for review)

**Comment by hirasso** [comment]
Yes, I have run `npm update` :)

**Review by daun — DISMISSED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
-				this.swup.log(`Matomo pageview (url '${url}').`);
-			} else {
-				console.warn('Matomo is not loaded.');
-			}
-		})
+		this.swup.on('contentReplaced', this.onContentReplaced);
+	}
+
+	/**
+	 * Remove event handlers on mount
```
Comment (gmrchk) [inline src/index.js]:
```suggestion
	 * Remove event handlers on unmount
```

File: src/index.js [hunk 2]
```diff
@@ -1,22 +1,52 @@
...
+	 * @returns {void}
+	 */
+	onContentReplaced = (popstate) => {
+		this.maybeTrackPageView();
+	};
+
+	/**
+	 * Tracks a page view to matomo, if it is installed
+	 */
+	maybeTrackPageView() {
```
Comment (gmrchk) [inline src/index.js]: IMO the `maybe` is not needed.

**Review by gmrchk — DISMISSED** [review]

File: src/index.js (hunk 2 above)
Comment (hirasso) [reply src/index.js]: I don't care much enough to discuss this ;) ...changed the function name to `trackPageView`

**Review by jensdraserschieb — DISMISSED** [review]

**Review by jensdraserschieb — APPROVED** [review]

**Comment by hirasso** [comment]
@gmrchk would you mind merging this? Also, happy holidays! 🎄🌟

**Pull request merged**